### PR TITLE
Chad/fix scrolling

### PR DESCRIPTION
**Pull request by matthewberry**

### CODE REVIEW

File: src/app/components/modal/modal.component.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
   </div>
 
   <div class="modal_footer">
-    <div class="button button_plain" (click)="close();$event.stopPropagation();">cancel</div>
-    <div class="button" (click)="loadDataSet();$event.stopPropagation();">Launch</div>
+    <div class="button button_plain" (click)="close();$event.stopPropagation();">Cancel</div>
+    <div class="button disabled" (click)="loadDataSet();$event.stopPropagation();">Launch</div>
```
Comment (matthewberry) [inline src/app/components/modal/modal.component.html:46]: @cholson how'd you want the `disabled` class applied for the Launch button?

File: src/app/components/node/node.component.ts [hunk 2]
```diff
@@ -39,9 +40,9 @@ export class NodeComponent implements OnInit, OnChanges, OnDestroy {
   gradientId = 'uninitialized';
 
   summaryColumns = [
+    { label: 'Type', cssSuffix: 'type', accessor: (severity: Severity, info: VariantInfo) => 'SNV' },
```
Comment (matthewberry) [inline src/app/components/node/node.component.ts:43]: @cholson are we sure we want Type first?

File: src/app/components/page/landing-page/landing-page.component.html [hunk 3]
```diff
@@ -86,10 +86,11 @@ <h2>Documentation</h2>
...
+-->
+    <p>PhyloDiver is developed by the <a href="http://vi-bio.ncsa.illinois.edu/" target="_blank">Visual Analytics Group</a> and <a href="https://www.el-kebir.net/" target="_blank">El-Kebir Lab</a> at the <a href="https://illinois.edu/" target="_blank">University of Illinois at Urbana-Champaign</a>, is supported by the <a href="https://cancer.illinois.edu/" target="_blank">Cancer Center of Illinois</a>, and is an open source project available on <a href="https://github.com/ncsa/phylodiver-viz" target="_blank">GitHub</a>. <span class="nowrap">Contact <a href="mailto:visualanalytics@illinois.edu">visualanalytics@illinois.edu</a></span></p>
```
Comment (matthewberry) [inline src/app/components/page/landing-page/landing-page.component.html:94]: @cholson branding question, here and below: better to say "developed by Visual Analytics and the El-Kebir Lab"?

File: src/app/components/modal/modal.component.html (hunk 1 above)
Comment (cholson) [reply src/app/components/modal/modal.component.html:46]: from charles… "Make Launch unclickable (grayed out) if your dataset selected and a file is not uploaded and validated"

File: src/app/components/node/node.component.ts (hunk 2 above)
Comment (cholson) [reply src/app/components/node/node.component.ts:43]: yep. this was a request from charles

File: src/app/components/page/landing-page/landing-page.component.html (hunk 3 above)
Comment (cholson) [reply src/app/components/page/landing-page/landing-page.component.html:94]: hmm, not sure i feel strongly either way…leave it for now?

File: src/app/components/modal/modal.component.html (hunk 1 above)
Comment (matthewberry) [reply src/app/components/modal/modal.component.html:46]: cool, thanks, I can do that

File: src/app/components/node/node.component.ts (hunk 2 above)
Comment (matthewberry) [reply src/app/components/node/node.component.ts:43]: thanks, and sorry, I see that now.

I'm double-checking with him that his preference remains the same even after we've added the severity column

File: src/app/components/page/landing-page/landing-page.component.html (hunk 3 above)
Comment (matthewberry) [reply src/app/components/page/landing-page/landing-page.component.html:94]: 10-4

**Pull request merged**